### PR TITLE
change texts

### PR DESCRIPTION
**Pull request by malviyagaurav**
- Fixes https://app.clickup.com/t/37308523/PRD-751
- Fixes https://app.clickup.com/t/37308523/PRD-1406
- Fixes https://app.clickup.com/t/37308523/PRD-1145
- Fixes https://app.clickup.com/t/37308523/PRD-469
- Fixes https://app.clickup.com/t/37308523/PRD-875

### CODE REVIEW

File: src/constant_texts.c [hunk 1]
```diff
@@ -408,7 +408,7 @@ const char* ui_text_incompatible_card_version = "Incompatible card version! Cont
 const char* ui_text_card_error_contact_support = "Contact support & share the device logs";
 const char* ui_text_device_and_card_not_paired = "This device & card are not paired";
 const char* ui_text_security_conditions_not_met = "Security conditions not met!";
-const char* ui_text_process_reset_due_to_inactivity = "Process was reset due to inactivity";
+const char* ui_text_process_reset_due_to_inactivity = "Operation aborted due to inactivity";
```
Comment (ujjwal-cyph) [inline src/constant_texts.c:411]: aborted does not sound good.

```suggestion
const char* ui_text_process_reset_due_to_inactivity = "Operation stopped due to inactivity";
```

File: src/constant_texts.c [hunk 2]
```diff
@@ -74,13 +74,13 @@ const char* ui_text_options_old_wallet[] = {
 const char* ui_text_options_new_wallet[] = {
     "Create Wallet",
     "Generate New Wallet",
-    "Restore From Seed"
+    "Restore from Seed"
```
Comment (ujjwal-cyph) [inline src/constant_texts.c:77]:
```suggestion
    "Restore from seed"
```

File: src/constant_texts.c [hunk 3]
```diff
@@ -408,7 +408,7 @@ const char* ui_text_incompatible_card_version = "Incompatible card version! Cont
 const char* ui_text_card_error_contact_support = "Contact support & share the device logs";
 const char* ui_text_device_and_card_not_paired = "This device & card are not paired";
 const char* ui_text_security_conditions_not_met = "Security conditions not met!";
-const char* ui_text_process_reset_due_to_inactivity = "Process was reset due to inactivity";
+const char* ui_text_process_reset_due_to_inactivity = "Operation aborted due to inactivity";
 const char* ui_text_device_already_provisioned = "Device already provisioned";
 const char* ui_text_check_cysync_app = "Check the CySync App";
```
Comment (ujjwal-cyph) [inline src/constant_texts.c:413]:
```suggestion
const char* ui_text_check_cysync_app = "Check the CySync app";
```

File: src/constant_texts.c [hunk 4]
```diff
@@ -178,7 +178,7 @@ const char* ui_text_authentication_required = "Device authentication required";
 const char* ui_text_start_auth_from_CySync = "Authenticate device from the CySync App";
```
Comment (malviyagaurav) [inline src/constant_texts.c:178]:
```suggestion
const char* ui_text_start_auth_from_CySync = "Authenticate device from the CySync app";
```

File: src/constant_texts.c [hunk 5]
```diff
@@ -258,7 +258,7 @@ const char* ui_text_confirm_pin = "Confirm PIN";
 const char* ui_text_word_hash = "Word # ";
 const char* ui_text_enter_word_hash = "Enter Word # %d";
 const char* ui_text_verify_word_hash = "Verify Word # ";
-const char* ui_text_is_word = "Is Word #%d\n%s?";
+const char* ui_text_is_word = "Is Word #%d\n%s";
```
[the remark this answers is not in the document]
Comment (malviyagaurav) [reply src/constant_texts.c:261]: yes

**Pull request merged**